habit/src/components/screens/login/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, Text } from 'react-native'
+import CustomTextInput from '../../TextInput'
+
+export default function Login(props){
+    let input
+    return(
+        <View>
+            <Text>Hello login form</Text>
+            <CustomTextInput 
+                style={{flex: 1}}
+                placeholder = 'Username'
+                ref={value => input = value}
```

Comment:
What will we use the ref for?
Why not just store it into the state?

Reply:
Or will we use hooks?

Reply:
ref is easy for these kind of small changes like input